Data/Samples/SampleGame/TypeScript/ez/AllComponents.ts

```diff
@@ -324,6 +324,8 @@ export class DebugTextComponent extends Component
   set Value3(value: number) { __CPP_ComponentProperty_set(this, 1816166542, value); }
   get Color(): Color { return __CPP_ComponentProperty_get(this, 1881354469); }
   set Color(value: Color) { __CPP_ComponentProperty_set(this, 1881354469, value); }
+  get MaxDistance(): number { return __CPP_ComponentProperty_get(this, 1272349630); }
```

Comment:
Remove typescript stuff?

Reply:
That's the sample game project. For now it still uses TS, but once I've added array support, it will be replaced as well.